README.md

```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# Litmus E2E Dashboard
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Prerequisites
 
-## Available Scripts
+- Install Yarn globally
 
-In the project directory, you can run:
+## Setup and Installation
 
-### `yarn start`
+- Clone the `gh-pages` branch of `litmus-e2e` repository
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+```
+  git clone -b gh-pages git@github.com:litmuschaos/litmus-e2e.git
+```
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+- Install all the required dependencies with the below command
 
-### `yarn test`
+```
+  yarn
```

Comment:
should it be yarn --help?

Reply:
The command `yarn` is an alias for the command `yarn install`. Will change it to `yarn install` for more clarity.